game/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-## 0.3.0
+## 0.3.1
 
-- Order weapons/shields by attack/defense
-- Small ui improvements
-- Add battle clock
-- Add "Battle Again" button
-- Limit battle logs size
-- Change combat formulas
+- Shop functionality
+- Create a color palette
+- Add css variables for colors
+- Add default font family
+- Add .editorconfig file to prevent diffs from indentation
+- Create a default style for buttons
+- Add new js files using 'js modules'
```

Comment:
The idea for the `CHANGELOG.md` is to keep the history. So you shouldn't delete the old and should do something like this instead:
```
## 0.3.0

- Order weapons/shields by attack/defense
- Small ui improvements
- Add battle clock
- Add "Battle Again" button
- Limit battle logs size
- Change combat formulas

## 0.4.0

- Shop functionality
- Create a color palette
- Add css variables for colors
- Add default font family
- Add .editorconfig file to prevent diffs from indentation
- Create a default style for buttons
- Add new js files using 'js modules'
```
By the way, the Shop functionality is a big deal! I think this deserves a "minor" version upgrade of the game, so from `0.3.0` to `0.4.0` and not `0.3.1` :smile: